bonsai/sdk/Cargo.toml

```diff
@@ -13,6 +13,7 @@ reqwest = { version = "0.11", default-features = false, features = [
   "json",
   "rustls-tls",
 ] }
+risc0-groth16 = { workspace = true }
```

Comment:
We depend on `bonsai-sdk` in the `risc0-zkvm` crate as a option prover backend, this wont create a circular dep right?

Reply:
Yes that is exactly why I left the sdk to depend directly on the `risc0-groth16` crate rather then using the new re-export from the `risc0-zkvm`. Basically both risc0-zkvm and bonsai-sdk depend on `risc0-groth16`, so we brake the circularity.